common/usb_imx.c

```diff
@@ -409,12 +412,15 @@ int count_sysprogs(char *initrd, char *console, char *append, int *sysprogs_sz)
 	return cnt;
 }
 
-int append_sysprogs(void *image, char *initrd, char *console, char *append, syspage_t *syspage, size_t offset, unsigned int addr)
+static int append_sysprogs(void *image, char *initrd, char *console, char *append, syspage_t *syspage, size_t offset, unsigned int addr)
 {
 	int i = 0, j = 0, pfd, cnt;
 	char *progs, *prog;
 
-	asprintf(&progs, "%s %s %s", console ? console : " ", initrd ? initrd : " ",  append ? append : " ");
+	if(asprintf(&progs, "%s %s %s", console ? console : " ", initrd ? initrd : " ",  append ? append : " ") < 0) {
```

Comment:
No big deal, but space is missing between `if` and `(`

Reply:
... and corrected other occurrences.